code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/load/unifiers/Unifier.kt

```diff
@@ -0,0 +1,8 @@
+package org.jetbrains.research.code.submissions.clustering.load.unifiers
+
+import org.jetbrains.research.code.submissions.clustering.model.Submission
+
+interface Unifier {
+    fun Submission.unify(): Submission
+    fun clear()
```

Comment:
Please, add a default empty implementation. It will help to avoid adding them if it is not necessary